packages/vaex-ml/vaex/ml/transformations.py

```diff
@@ -573,3 +573,65 @@ def transform(self, df):
                 expr = expr / self.scale_[i]
             copy[name] = expr
         return copy
+
+
+@register
+@generate.register
+class GroupByTransformer(Transformer):
+    '''The GroupByTransformer creates aggregations via the groupby operation, which are
```

Comment:
""" is default for docstrings

Reply:
None of the others have it tho.. i was just following what we had so far.. 

Should I change everything?

Reply:
lets keep it then